.golangci.yml

```diff
@@ -0,0 +1,49 @@
+run:
+  timeout: 5m
+  allow-parallel-runners: true
+
+issues:
+  # don't skip warning about doc comments
+  # don't exclude the default set of lint
+  exclude-use-default: false
+  # restore some of the defaults
+  # (fill in the rest as needed)
+  exclude-rules:
+    - path: "api/*"
+      linters:
+        - lll
+    - path: "internal/*"
+      linters:
+        - dupl
+        - lll
+linters:
+  disable-all: true
+  enable:
```

Comment:
It can be good to have, but with all scanners we already have, I'm not sure it will be a worthwhile addition.

Also, I prefer that security scan results are private. I'm unsure if adding it here would lead to making the warnings more publicly visible.

Reply:
we can try to see, IIRC KServe upstream does use it.

Reply:
Hmm... I didn't know.
OK, I added it.

Reply:
lets do it on a upcoming PR then, I can handle it later :)

Reply:
Logged here: https://issues.redhat.com/browse/RHOAIENG-17149